src/styles/main.scss

```diff
@@ -1 +1,84 @@
-// put code here
+.body {
+  margin: 0;
+  display: flex;
+  justify-content: center;
+  height: 100vh;
+  align-items: center;
+}
+
+
+.stopwatch {
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__center {
+    box-sizing: border-box;
+    position: relative;
+    height: 5vmin;
+    width: 5vmin;
+    border-radius: 100%;
+    background-color: #f6a603;
+  }
+
+  &__seconds-hand {
+    position: absolute;
+    height: 38vmin;
+    width: 1.5vmin;
+    background-color: #2c8000;
+    bottom: 50%;
+    left: 50%;
+    animation: rotate 60s linear infinite running;
+    transform-origin: bottom center;
+    z-index: -1;
```

Comment:
it's better to use `positive` values (1+)

Reply:
When I use positive values my hands are shown: 

<img width="447" alt="Screenshot 2024-01-30 at 10 23 33 PM" src="https://github.com/mate-academy/layout_stop-watch/assets/148825530/7070ce90-fba7-49ba-a2fa-48034710e9f2"